components/telegram_bot_api/actions/send-album/send-album.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable camelcase */
+const telegram_bot_api = require("../../telegram_bot_api.app.js");
+
+module.exports = {
+  key: "telegram_bot_api-send-album",
+  name: "Send an Album (Media Group)",
+  description: "Sends a group of photos or videos as an album",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    telegram_bot_api,
+    chatId: {
+      propDefinition: [
+        telegram_bot_api,
+        "chatId",
+      ],
+    },
+    media: {
+      type: "any",
+      label: "Media",
+      description: "A JSON-serialized array describing photos and videos to be sent, must include 2–10 items (e.g., `[{\"type\":\"photo\",\"media\":\"https://example.com/myImage.jpeg\"},{\"type\":\"video\",\"media\":\"/tmp/myVideo.mp4\"}]`)",
```

Comment:
In addition to the description I think is also necessary the link to the docs

Reply:
Added a link to the docs:

https://github.com/PipedreamHQ/pipedream/blob/c80b5afcf9851debaa7cab8efab46a3619aeaa1f/components/telegram_bot_api/actions/send-album/send-album.mjs#L22-L27